Makefile

```diff
@@ -1,12 +1,15 @@
 PROJECT_NAME := "poseidon"
-PKG := "github.com/openHPI/$(PROJECT_NAME)/cmd/$(PROJECT_NAME)"
+REPOSITORY_OWNER = "openHPI"
+PKG := "github.com/$(REPOSITORY_OWNER)/$(PROJECT_NAME)/cmd/$(PROJECT_NAME)"
 UNIT_TESTS = $(shell go list ./... | grep -v /e2e)
 
 DOCKER_E2E_CONTAINER_NAME := "$(PROJECT_NAME)-e2e-tests"
 DOCKER_TAG := "poseidon:latest"
 DOCKER_OPTS := -v $(shell pwd)/configuration.yaml:/configuration.yaml
-# don't use :latest to prevent Nomad from trying to pull the image automatically
-E2E_TEST_DOCKER_IMAGE = "poseidon/e2e-docker-image:ci"
+LOWER_REPOSITORY_OWNER = $(shell echo $(REPOSITORY_OWNER) | tr A-Z a-z)
+REGISTRY = "ghcr.io"
+TAG = ":main"
```

Comment:
Is this tag ok or should we investigate how and why Github is automatically using this tag?

Reply:
To be honest: I don't mind, that's fine for me (at least when it works as intended with the tests). I know that usually, Docker uses the `latest` tag but GitHub defaults to the branch name (which is `main`). 

From my perspective, we don't need to continue investigating that any further.